### DISCUSSION TITLE
[API] Received hardcore achievement count missing in API_GetUserRecentlyPlayedGames.php

### DISCUSSION THREAD

**Discussion by DFelten** [RetroAchievements/RAWeb#1037] [Ideas]
I'm currently working on an app for RetroAchievements and would need the number of received hardcore achievements in the list of recently played games (`API_GetUserRecentlyPlayedGames.php`). Unfortunately this is not possible at the moment.  I only get the number of casual achievements (`NumAchieved`).

The endpoint `API_GetGameInfoAndUserProgress.php` provides the number of hardcore achievements, so I can display and filter them on the game page. Actually, I intended that the user can set whether to show the number of hardcore or normal achievements. Unfortunately, this is not possible for me in the game list.

Would it be possible to output this number as well?

**Comment by Jamiras** [comment]
Implemented in #1038

**Reply by DFelten** [reply]
Great, thanks! When will the change be deployed so that the API will return the number?

**Reply by luchaos** [reply]
Released and deployed: https://github.com/RetroAchievements/RAWeb/releases/tag/1.89.0
